table.cc

```diff
@@ -2354,8 +2354,12 @@ future<> table::update_repair_time(dht::token_range range, gc_clock::time_point
 
 gc_clock::time_point
 table::get_repair_based_tombstone_gc_before(const dht::decorated_key& dk) const {
-    // FIXME: Make propagation_delay tunable
-    constexpr auto propagation_delay = std::chrono::seconds(3600);
+    const auto& options = _schema->tombstone_gc_options();
+    if (options.gc_disabled()) {
+        tlogger.trace("Get repair based gc time for ks={}, table={}, dk={}, gc disabled", schema()->ks_name(), schema()->cf_name(), dk);
+        return gc_clock::time_point::min();
```

Comment:
Should we also add an option for the old behavior?

Reply:
I think it's a good idea to have a way too fallback until the new behavior has been proven successful for everyone.

Reply:
In my latest version that I am working on I added too more modes. One of them is for the old behavior.

Reply:
> You mean in the version which is yet to be pushed? Not in the current version of this PR?

A new version I will publish soon.